round.py

```diff
@@ -137,11 +137,13 @@ async def challenge(self, ctx, *users: discord.Member):
         if len(users) == 0:
             await ctx.send(f"The correct usage is `.round challenge @user1 @user2...`")
             return
-        if ctx.author not in users:
-            users.append(ctx.author)
-        if [ctx.author] == users:
+        
+        if ctx.author in users:
             await ctx.send(f"You cannot compete against yourself!")
             return
```

Comment:
Suppose a user does .round challenge @himself @opponent, the bot will say that the user is challenging himself. The bot should say this when the challenger is the only person in the list

Reply:
Also self challenging should be a feature now since it does not affect the delta anyways. The bot should just add an extra line in the embed where the users have to react that this is a practice round and your delta wont be affected

Reply:
Fair, but maybe a separate command for self challenging for practice would be a better idea.

Reply:
The features will be exactly the same so making a new command would be redundant